### ISSUE TITLE
'BTCardCapabilities' is deprecated: The UnionPay SMS integration is deprecated, as UnionPay can now be processed as a credit card through their partnership with Discover. Use `BTCardClient.tokenizeCard(card: completion:)`.

### ISSUE THREAD

**Issue by yagnyam** [braintree/braintree-ios-drop-in#385]
**Integration Details (please complete the following information):**
 - SDK/Library Version: [e.g. 9.0.0]
 - Environment: [e.g. Sandbox or Production]
 - iOS Version: [e.g. iOS 13.0]
 - Xcode Version: [e.g. Xcode 13.2]
 - Device [e.g. iPhone 12]
 - Integration type & version [e.g. CocoaPods 1.1.1, Swift Package Manager 5.1]
 - Development Processor Type: [e.g. Intel, M1/ Apple Silicon]

**Describe the bug**
Description of what the bug is. Please include as many details as possible.

**To Reproduce**
3. See error

**Expected behavior**
Description of what you expected to happen.

**Screenshots**

**Comment by Moe-kanan** [comment]
Same issue here. We're using Expo React Native app and Bitrise to build.

**Comment by mattddean** [comment]
@Moe-kanan If you have a way to lock the Braintree pod which this pod depends on at 5.17.0, that's a workaround for now, since Braintree [5.18.0](https://cocoapods.org/pods/Braintree#5180-2022-12-13) has a that causes this issue. 

On a side note, I'm also using Expo. I'm using react-native-braintree-dropin-ui and have had to make some significant [patches](https://www.npmjs.com/package/patch-package) to it to get things working. Maybe we could compare notes sometime.

```podspec
s.dependency    'BraintreeDropIn', '9.7.0'
s.dependency    'Braintree', '5.17.0'
s.dependency    'Braintree/DataCollector', '5.17.0'
```

**Comment by jaxdesmarais** [comment]
Hello everyone - 

The fix for this was just released in version [9.8.0 of the Drop-in SDK](https://github.com/braintree/braintree-ios-drop-in/releases/tag/9.8.0). Once you've had a chance to update, please let us know if you are still running into errors. Thanks again for bringing this to our attention!

**Comment by dimaP8889** [comment]
Hi, I have update BraintreeDropIn to 9.8.0 and Braintree to 5.19.0 and still got this issue

**Comment by scannillo** [comment]
>  Hi, I have update BraintreeDropIn to 9.8.0 and Braintree to 5.19.0 and still got this issue

Hi @dimaP8889 - can you provide more detail / some screenshots on how the issue is still manifesting?

**Comment by scannillo** [comment]
Closing due to inactivity. This fix was released in [9.8.0 of the Drop-in SDK](https://github.com/braintree/braintree-ios-drop-in/releases/tag/9.8.0).

**Comment by Justin-Credible** [comment]
I was also still experiencing this issue on the following versions:

* `BraintreeDropIn` => `9.8.1`
* `Braintree` => `5.21.0`

The workaround I used was to set `GCC_WARN_ABOUT_DEPRECATED_FUNCTIONS` XCode build setting on the `BraintreeDropIn` target of the Pods project.

I did this by adding the following to the `post_install` of the Podfile:

```
  post_install do |installer|
    # Set GCC_WARN_ABOUT_DEPRECATED_FUNCTIONS = 'NO' for BraintreeDropIn Pod to work around build deprecation warnings.
    noDeprecationWarningTargets = ['BraintreeDropIn']
    installer.pods_project.targets.each do |target|
      if noDeprecationWarningTargets.include? target.name
        target.build_configurations.each do |config|
          config.build_settings['GCC_WARN_ABOUT_DEPRECATED_FUNCTIONS'] = 'NO'
        end
        puts "GCC_WARN_ABOUT_DEPRECATED_FUNCTIONS was set to NO for #{target.name}"
      end
    end
  end
```

Since the Podfile is not in source control, I made this modification from my Expo module/plugin script, which is applied during the `expo prebuild` step.
